### DISCUSSION TITLE
Resources for Now + Auth0

### DISCUSSION THREAD

**Discussion by alecj1240** [vercel/vercel#4119] [General]
Hey, I'm really hoping to combine Auth0 and Now, and would love to know if anyone has any examples or resources they could share. Any language would be helpful, but especially Go!

**Comment by coetry** [comment]
Hi @alecj1240 We don't currently have any examples for Go (we welcome community contributions :smile: ), but we have a great [example for Next.js](https://github.com/zeit/next.js/tree/canary/examples/auth0) that has been updated recently!
